### PR TITLE
Add a test which creates a user with :mailing

### PR DESCRIPTION
**Pull request by ericnormand**
https://www.pivotaltracker.com/story/show/103374528

We want to make sure that we can pass in all types of addresses,
including a mailing address. I duplicated the basic user creation test
and added a mailing address along with the registered address. It passed
the first time. I also made sure it was being passed in the user.create
event.

### CODE REVIEW

File: test/user_works/handlers_test.clj [hunk 1]
```diff
@@ -37,11 +37,44 @@
...
-     (testing "adds a message to the create events channel"
-       (async/alt!!
-         (async/timeout 1000) (is false "No create event received after 1 second.")
-         creates-chan ([event] (is (= (get-in resp [:user :id])
-                                      (:id event))))))))
+      (testing "adds a message to the create events channel"
+        (async/alt!!
+          (async/timeout 1000) (is false "No create event received after 1 second.")
+          creates-chan ([event]
+                        (is (= (get-in resp [:user :id])
```
Comment (cap10morgan) [inline test/user_works/handlers_test.clj]: Is this a redundant check now?

File: test/user_works/handlers_test.clj [hunk 2]
```diff
@@ -37,11 +37,44 @@
...
+                                :zip "70119"
+                                :ocd-divisions
+                                #{"ocd-division/country:us/state:la"}}
+                               :mailing
+                               {:street "2659 Iberville St."
+                                :city "New Orleans"
+                                :state "LA"
+                                :zip "70119"
+                                :ocd-divisions
+                                #{"ocd-division/country:us/state:la"}}}}
```
Comment (cap10morgan) [inline test/user_works/handlers_test.clj]: Should we use a different address here to make sure the system is storing them separately?

Comment (ericnormand) [inline test/user_works/handlers_test.clj]: Good idea.

**Comment by cap10morgan** [comment]
Assuming test run is green, LGTM.

**Pull request merged**